Makefile

```diff
@@ -53,7 +53,7 @@ OPM=$(shell pwd)/_cache/opm
 KUSTOMIZE_VERSION ?= v5.5.0
 KUSTOMIZE=$(shell pwd)/_cache/kustomize
 KIND ?= $(shell pwd)/_cache/kind
-KIND_VERSION=v0.23.0
+KIND_VERSION ?= v0.29.0
```

Comment:
will this add a strict dependency with the k8s version?
I thought networkpolicies were around for long time. What's specific in this bump?

Reply:
Network policies were added in 0.24,  https://github.com/kubernetes-sigs/kind/releases/tag/v0.24.0

I do not think having the latest kind adds a strict dependency, we might get warnings if we do API that are getting deprecated. I think is okay that bump as long as CI is green, but I can also bump to any version >=0.24